mods/rv/rules/ai.yaml

```diff
@@ -568,7 +573,7 @@ Player:
 		PowerTypes: gapowr, napowr, nanrct, yapowr
 		BarracksTypes: gapile, nahand, yabrck
 		VehiclesFactoryTypes: gaweap, naweap, yaweap
-		ProductionTypes: gapile, nahand, yabrck, gaweap, naweap, yaweap, gaairc, yadisk #, naheli
+		ProductionTypes: gapile, nahand, yabrck, gaweap, naweap, yaweap, gaairc, yadisk, naheli
```

Comment:
Small nit, can you switch yadisk and naheli around so faction order is Allies, Soviets, Yuri?

Reply:
fixed